_syllabus/seed.py

```diff
@@ -76,12 +77,19 @@
         hwDate  = lecDate.replace(hours=15)
         solDate = hwDate.shift(weeks=1)
 
+
+	 
+	oh_plus_plus_link = wk{0:02d}-ohpp.md'.format(idx + 1)
+
```

Comment:
How about having the OH++ notes mirror [this pattern][] from the homework release where the line only shows if the yaml key is present?  You can add it to `seed.py` commented out so we don't forget about it, but also so Travis stays happy.

[this pattern]: https://github.com/c4cs/c4cs.github.io/blob/6e6a24810628213b15c7221589544bbf6da0bdeb/_layouts/syllabus.html#L91-L114